movingpandas/tests/test_cpa.py

```diff
@@ -0,0 +1,467 @@
+import datetime
+import pathlib
+import unittest.mock
```

Comment:
Do we need to mix pytest and unittest?

Reply:
I used the mock functionality of unittest (builtin) and pytest for the fixtures.